src/System.Net.HttpListener/src/System/Net/HttpHeaderStrings.Unix.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net
+{
+    internal class HttpHeaderStrings
+    {
+        internal const string Authorization = "Authorization";
+        internal const string ContentType = "Content-Type";
+        internal const string Charset = "charset=";
+        internal const string Server = "Server";
+        internal const string NetCoreServerName = ".NETCore-HTTPAPI/1.0";
```

Comment:
It's used in sending http headers, sends, Server=value. Replaced Mono with .NETCore.